sagemaker_model_monitor/fairness_and_explainability_jsonlines/SageMaker-Monitoring-Feature-Attribution-Drift-for-Endpoint.ipynb

```diff
@@ -0,0 +1,1589 @@
+{
```

Comment:
<p>Where does the dataset come from? Maybe you should mention it here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Thanks. The two notebooks use the same data/model from another Clarify notebook, mentioned at the beginning of the section. A trained model is used to reduce the runtime of the notebook, considering that it already takes long for model monitor to trigger an execution.</p><p> </p><blockquote>This example includes a prebuilt <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/linear-learner.html" target="_blank">SageMaker Linear Learner</a> model trained by <a href="https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_jsonlines_format.ipynb" target="_blank">a SageMaker Clarify offline processing example notebook</a>. </blockquote>

Reply:
Per offline discussion, revise the cell to mention the data source. Thanks.

> This example includes two dataset files, both in the JSON Lines format. The data also originates from [the SageMaker Clarify offline processing example notebook](https://github.com/aws/amazon-sagemaker-examples/blob/main/sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_jsonlines_format.ipynb).